.pre-commit-config.yaml

```diff
@@ -9,20 +9,20 @@ default_language_version:
   python: python3.10
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.5.0
+    rev: v4.6.0
     hooks:
       - id: trailing-whitespace
       - id: end-of-file-fixer
       - id: check-yaml
       - id: check-added-large-files
   - repo: https://github.com/compilerla/conventional-pre-commit
-    rev: v3.0.0
+    rev: v3.3.0
     hooks:
       - id: conventional-pre-commit
         stages: [commit-msg]
         args: []
   - repo: https://github.com/psf/black
-    rev: 23.12.1
+    rev: 24.4.2
```

Comment:
There was a breaking change in pre-commit making this config obsolete, so I had to run `pre-commit autoupdate` :sweat: